AppSizer/AppSizer/AppDelegate.swift

```diff
@@ -18,15 +18,17 @@
 //
 
 import UIKit
-import TwilioVerify
+import TwilioVerifySDK
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
   var window: UIWindow?
 
   func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-    print(TwilioVerifyVersionNumber)
+    if let sdkVersion = Bundle(for: TwilioVerifyBuilder.self).infoDictionary?["CFBundleShortVersionString"] {
```

Comment:
Why was this changed? We experienced an issue with reading the SDK version because it was not present the SDK value in a consumer app, it returned the app version, was this fixed? See https://github.com/twilio/twilio-verify-ios/pull/121 and Alamofire for example: https://github.com/Alamofire/Alamofire/blob/master/Source/Alamofire.swift#L29